src/Uno.UI.RemoteControl/Messages/ProcessorRemoval.cs

```diff
@@ -0,0 +1,21 @@
+namespace Uno.UI.RemoteControl.Messages
+{
+	public class ProcessorRemoval : IMessage
+	{
+		public const string Name = nameof(ProcessorRemoval);
+
+		public ProcessorRemoval(string path, string processorScope)
+		{
+			Path = path;
+			ProcessorScope = processorScope;
+		}
+
+		public string Scope => "RemoteControlServer";
```

Comment:
Can scopes/messages that are shared between client and server be in a single file that's shared between the projects?

Reply:
> Can scopes/messages that are shared between client and server be in a single file that's shared between the projects?

Do you mean the constant string value(s)?